config/seata-config-core/src/main/resources/registry.conf

```diff
@@ -66,6 +66,7 @@ config {
     serverAddr = "127.0.0.1:2181"
     sessionTimeout = 6000
     connectTimeout = 2000
+    serializer = ""
```

Comment:
It is not recommended that the serializer be exposed to the user to increase the cost of understanding what code is actually defined for the configuration item.

Reply:
This suggestion is very good.